app/src/main/java/com/github/swent/echo/compose/components/HomeScreen.kt

```diff
@@ -60,6 +60,7 @@ fun HomeScreen(navActions: NavigationActions, homeScreenViewModel: HomeScreenVie
             Content(paddingValues, navActions, homeScreenViewModel)
         }
     }
+    ConnectivityStatus()
```

Comment:
We can remove this for now, we will add it back to the content of the home screen once we have the `NetworkService` implemented.